binr/rax2/rax2.c

```diff
@@ -228,8 +229,60 @@ static int rax(char *str, int len, int last) {
 		return true;
 	}
 	if (flags & (1 << 2)) { // -S
-		for (i = 0; i < len; i++) {
-			printf ("%02x", (ut8) str[i]);
+		for (int i = 0; i < len; i++) {
+			if (str[i] == '\\') {
+         
+				++i;
+				if (str[i] == 'n') printf("%02x", 10);
+				else if (str[i] == '\0') printf("%02x", 0);
+				else if (str[i] == 'a') printf("%02x", 7);
+				else if (str[i] == 'b') printf("%02x", 8);
+				else if (str[i] == 'f') printf("%02x", 12);
+				else if (str[i] == 'r') printf("%02x", 13);
+				else if (str[i] == 't') printf("%02x", 9);
+				else if (str[i] == 'v') printf("%02x", 11);
+				else if (str[i] == '\\') printf("%02x", 92);
+				else if (str[i] == '\'') printf("%02x", 39);
+				else if (str[i] == '\"') printf("%02x", 34);
+				else if (str[i] == '\?') printf("%02x", 63);
+				else if (str[i] == 'x') {
+					i++;
+					while (IS_HEXCHAR (str[i])) {
+						printf("%c", tolower (str[i]));
+						i++;
+					}
+					i--; 
+				} else if (IS_OCTAL (str[i])) {
```

Comment:
so many IS_OCTAL nested conditions, i dont see why is-octal should be checked for several bytes